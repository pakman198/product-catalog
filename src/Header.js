import React from 'react';

const Header = () => {
  return (
    <div className="ui borderless top fixed menu">
      <div className="item">
        <i className="large icon fly"></i>
        WestElm
      </div>
    </div>
  );
}

export default Header;