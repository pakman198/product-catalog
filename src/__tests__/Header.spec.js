import React from 'react'
import { render } from '@testing-library/react'

import Header from '../components/Header';

test('Header.js', () => {
  const { asFragment } = render(<Header />);
  
  expect(asFragment()).toMatchSnapshot();
});