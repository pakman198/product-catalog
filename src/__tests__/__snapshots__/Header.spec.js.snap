// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header.js 1`] = `
<DocumentFragment>
  <div
    class="ui borderless top fixed menu"
  >
    <div
      class="item"
    >
      <i
        class="large icon fly"
      />
      WestElm
    </div>
  </div>
</DocumentFragment>
`;
